deployment/docker-compose.yml

```diff
@@ -41,16 +39,40 @@ services:
     build:
       context: ../admin_app
       dockerfile: Dockerfile
+      args:
+        NEXT_PUBLIC_BACKEND_URL: ${NEXT_PUBLIC_BACKEND_URL}
     command: >
       npm start
     depends_on:
       - core_backend
     restart: always
     env_file:
       - .env
+  nginx:
+    image: nginx:1.25-alpine
+    restart: always
+    depends_on:
+      - core_backend
+      - admin_app
     ports:
-      - "3000:3000"
-
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ./data/nginx:/etc/nginx/conf.d
+      - ./data/certbot/conf:/etc/letsencrypt
+      - ./data/certbot/www:/var/www/certbot
+    command: >
+      /bin/sh -c "envsubst '$$\{DOMAIN\}' < /etc/nginx/conf.d/app.conf.template > /etc/nginx/conf.d/app.conf
```

Comment:
`envsubst` replaces the ${DOMAIN} in the `app.conf.template` file and generates `app.conf`

Reply:
Same as previous comment - perhaps keep a note of this somewhere permanently?

Reply:
It's detailed `dev` stuff for if you are making changes to the reverse proxy.
We don't need to put it in the public docs but good for us to know.

Reply:
I added comment here for you as a reviewer but not needed for the masses